tests/start.go

```diff
@@ -34,13 +35,32 @@ func Start(o *option.Option) {
 			containerShouldBeRunning(o, testContainerName)
 		})
 
-		for _, attach := range []string{"--attach", "-a"} {
+		for _, attach := range []string{"--attach", "-a", "-a=true", "--attach=true"} {
 			attach := attach
 			ginkgo.It(fmt.Sprintf("with %s flag, should start the container with stdout", attach), func() {
 				command.Run(o, "create", "--name", testContainerName, localImages[defaultImage], "echo", "foo")
 				output := command.StdoutStr(o, "start", attach, testContainerName)
 				gomega.Expect(output).To(gomega.Equal("foo"))
 			})
 		}
+
+		ginkgo.It("should run a container without an init process when --init=false flag is used", func() {
+			command.Run(o, "run", "--name", testContainerName, "--init=false", localImages[defaultImage], "ps", "aux")
```

Comment:
nit: Might be a little clearer to do something like `"ps", "ao", "pid,command"` so that you only get the columns you need from the output